src/common/options/mds.yaml.in

```diff
@@ -211,56 +212,56 @@ options:
 - name: mds_dir_keys_per_op
   type: int
   level: advanced
-  desc: number of directory entries to read in one RADOS operation
+  desc: Number of directory entries to read in one RADOS operation
   default: 16384
   services:
   - mds
   with_legacy: true
 - name: mds_decay_halflife
   type: float
   level: advanced
-  desc: rate of decay for temperature counters on each directory for balancing
+  desc: Rate of decay for temperature counters on each directory for balancing
   default: 5
   services:
   - mds
   with_legacy: true
 - name: mds_beacon_interval
   type: float
   level: advanced
-  desc: interval in seconds between MDS beacon messages sent to monitors
+  desc: Interval in seconds between MDS beacon messages sent to Monitors
   default: 4
   services:
   - mds
   with_legacy: true
 - name: mds_beacon_grace
   type: float
   level: advanced
-  desc: tolerance in seconds for missed MDS beacons to monitors
+  desc: Tolerance in seconds for missed MDS beacons to monitors
   fmt_desc: The interval without beacons before Ceph declares an MDS laggy
-    (and possibly replace it).
+    (and possibly replaces it).
   default: 15
   services:
   - mds
   with_legacy: true
 - name: mds_heartbeat_reset_grace
   type: uint
   level: advanced
-  desc: the basic unit of tolerance in how many circles in a loop, which will
-    keep running by holding the mds_lock, it must trigger to reset heartbeat
+  desc: The number of loop operations while hoding the mds_lock to tolerate
+    before triggering a heartbeat reset
```

Comment:
The existing wording seems confusing, please validate my phrasing.

Reply:
... wow that wording was bad. It should be instead:

`tolerance in seconds for long-running MDS operations which do not periodically reset the heartbeat timeout`